python/ray/function_manager.py

```diff
@@ -444,7 +444,14 @@ def get_execution_info(self, driver_id, function_descriptor):
         # the function from GCS.
         with profiling.profile("wait_for_function", worker=self._worker):
             self._wait_for_function(function_descriptor, driver_id)
-        return self._function_execution_info[driver_id][function_id]
+        try:
+            info = self._function_execution_info[driver_id][function_id]
```

Comment:
When could `info = self._function_execution_info[driver_id][function_id]` fail?

Reply:
Shouldn't this always succeed?

Reply:
It fails when the function that we are trying to call is not defined in the actor.

Reply:
@robertnishihara I have fixed this failure. This is because multiple drivers are exporting the same actor class with the same ID. Finally, there is only one entry in GCS with this ID. Other workers expect the actor imported using the expected driver, but the driver id is not correct. Therefore, the driver ID is wrong and causes KeyError.

Reply:
I have added the driver ID in actor key. I checked the remote function key. It has the driver id in it, too.

Reply:
Oh nice! Great catch!

Reply:
That's pretty subtle.